src/provider/CentralServerProvider.tsx

```diff
@@ -860,6 +898,10 @@ export default class CentralServerProvider {
     return tenant?.endpoint + '/v1/auth';
   }
 
+  private buildRestServerAPIURL(): string {
```

Comment:
Could you make that helper used everywhere needed?

Reply:
I dont understand

Reply:
You have introduced a private method helper to build URI to the API REST endpoints, you can generalise its usage everywhere in that file.

Reply:
Me, sure